### ISSUE TITLE
iOS video playback scaling

### ISSUE THREAD

**Issue by oxenreport** [oxen-io/session-ios#748]
scaling for videos during playback on iOS is problematic, iOS version unknown

**Comment by oxenreport** [comment]
additional comments: video remains zoomed into its playback mode and does not respond with an option to zoom out for proper media display

**Comment by DiCE1904** [comment]
Also, when you rotate your device, the video does not rotate nor scale

**Comment by burtonemily** [comment]
Hi there, thank you so much for taking the time to report this bug! We hate bugs as much as you do — reporting them helps make Session better for everyone. Our team is currently looking into this issue, we will keep you updated once we know more.

**Comment by KeeJef** [comment]
Yeah can reproduce here, widescreen media is not properly being scaled which means half of the media gets chopped off, would also be great to have rotation for widescreen media display

**Comment by mpretty-cyro** [comment]
We've reworked video playback to use native controls in #929 which should fix these issues

**Comment by mpretty-cyro** [comment]
@oxenreport @DiCE1904 Are either of you still experiencing these issues?
